test/envs/regional-dr.yaml

```diff
@@ -41,6 +42,8 @@ templates:
     container_runtime: containerd
     network: "$network"
     memory: "4g"
+    addons:
+      - metrics-server
```

Comment:
Do we need the metrics server on the hub?

What is the visible effect of adding the metrics server?

Reply:
It is very useful in analyzing the resource consumption of various pods on the node and will be required to make educated guesses on the resource limits that we configure on the ramen hub operator pod.

Reply:
I think this is also useful in testing some of the metrics feature of ramen that we have introduced recently.